src/components/pages/RoleSignup/Applications/Mentee.js

```diff
@@ -49,9 +50,11 @@ const initialFormValues = {
 
 const Mentee = ({ dispatch, error, successPage }) => {
   const { formValues, handleChange } = useForms(initialFormValues);
+  const { push } = useHistory();
 
```

Comment:
I love the use of useHistory, super easy to just send the user to the success page